addons/link_tracker/models/mail_render_mixin.py

```diff
@@ -38,6 +38,9 @@ def _shorten_links(self, html, link_tracker_vals, blacklist=None, base_url=None)
         short_schema = base_url + '/r/'
```

Comment:
A few typos in here:
reciptients -> recipients
an URL -> a URL
show an error -> shows an error
awa recommended I use a spell checker on my commits recently so I'll do the same, it's the one feature of vscode git integration that works :p

Reply:
+1 I had it on my previous laptop, forgot to set it up again, btw I use nvim :nerd_face: